docs/images.md

```diff
@@ -5,31 +5,157 @@ You create your Docker image and push it to a registry before referring to it in
 
 The `image` property of a container supports the same syntax as the `docker` command does, including private registries and tags.
 
+## Updating Images
+
+The default pull policy is `PullIfNotPresent` which causes the Kubelet to not pull an image if it already exists. If you would like to always force a pull you must set a pull image policy of `PullAlways` or specify a `:latest` tag on your image.
+
 ## Using a Private Registry
+Private registries may require keys to read images from them.
+Credentials can be provided in several ways:
+  - Per-cluster, automatic
+    - all pods can read the project's private registry
+    - only on GCE/GKE
+  - Per-node keys
```

Comment:
I don't see this expanded or described.  The gotcha about all nodes requiring credentials to all registries is important.